src/main/java/life/qbic/portal/portlet/OMEROClientPortlet.java

```diff
@@ -273,12 +273,12 @@ private Panel getImgViewer() {
             // Exceptions need to be handled here since they are event based and do not bubble up
             try {
                 return (Component) metadataButton(imageInfo.getImageId());
-            } catch (Exception e) {
+            }catch (Exception e) {
```

Comment:
```suggestion
            } catch (Exception e) {
```
Please check your IDE code formatting